src/@koop-components/common/alert/_alert.scss

```diff
@@ -58,11 +58,24 @@
   }
 
   &--fixed {
-    position: fixed;
-    top: 0;
+    position: absolute;
```

Comment:
Waarom absolute? Fixed is fijner op het moment dat je lager op de pagina die melding krijgt, toch?

Reply:
Goed punt, inderdaad. 👍